test/AppTest/Service/Speaker/Exception/MeetupNotFoundTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types = 1);
+
+namespace AppTest\Service\Speaker\Exception;
+
+use App\Service\Speaker\Exception\SpeakerNotFound;
+use Ramsey\Uuid\Uuid;
+
+/**
+ * @covers \App\Service\Speaker\Exception\SpeakerNotFound
+ */
+class SpeakerNotFoundTest extends \PHPUnit_Framework_TestCase
```

Comment:
![6256a2](https://cloud.githubusercontent.com/assets/496145/19267172/362d025a-8fa5-11e6-82eb-40cbbe5728a8.png)

seems to be - I don't think it updates the review here, have to look specifically at the diff